dot/types/grandpa.go

```diff
@@ -17,13 +17,13 @@
 package types
 
 import (
-	"encoding/binary"
 	"fmt"
 	"io"
 
 	"github.com/ChainSafe/gossamer/lib/common"
 	"github.com/ChainSafe/gossamer/lib/crypto/ed25519"
 	"github.com/ChainSafe/gossamer/lib/scale"
+	scale2 "github.com/ChainSafe/gossamer/pkg/scale"
```

Comment:
why do we need both scale packages?

Reply:
Well right now encoding/decoding for the grandpaVoter is using custom logic (since its not scale encoded I do not believe) and its uses the old scale to encode/decode. Dot/types is the integration I will finish after this one is merged so I was planning to fix it then but if you want it in this pr I can do that.

Reply:
As long as we have an issue for this that's fine.